### PR TITLE
KEYCLOAK-5973 Filtering of Gatling results

### PR DESCRIPTION
**Pull request by tkyjovsk**
Requires/includes commit for KEYCLOAK-5372.

### CODE REVIEW

**Comment by mstruk** [comment]
Needs to be rebased before continuing review - KEYCLOAK-5372 is now in master.

File: testsuite/performance/tests/src/test/scala/keycloak/AdminConsoleSimulation.scala [hunk 1]
```diff
@@ -99,4 +100,14 @@ class AdminConsoleSimulation extends Simulation {
...
+
+  after {
+    if (TestConfig.filterResults) {
+      new LogProcessor(getClass).filterLog(
+        TestConfig.measurementStartTime, 
+        TestConfig.measurementEndTime,
+        false, true, true)
+    }
+  }
+
```
Comment (mstruk) [inline testsuite/performance/tests/src/test/scala/keycloak/AdminConsoleSimulation.scala]: Maybe we could move this code to CommonScenarioBuilder and only have here something like:
```
after {
  filerResults(getClass)
}
```

File: testsuite/performance/tests/src/test/scala/keycloak/BasicOIDCSimulation.scala [hunk 2]
```diff
@@ -59,4 +60,13 @@ class BasicOIDCSimulation extends Simulation {
...
 
+  after {
+    if (TestConfig.filterResults) {
+      new LogProcessor(getClass).filterLog(
+        TestConfig.measurementStartTime, 
+        TestConfig.measurementEndTime,
+        false, true, true)
+    }
+  }
+
```
Comment (mstruk) [inline testsuite/performance/tests/src/test/scala/keycloak/BasicOIDCSimulation.scala]: Same here ...

File: testsuite/performance/tests/src/test/scala/keycloak/AdminConsoleSimulation.scala (hunk 1 above)
Comment (tkyjovsk) [reply testsuite/performance/tests/src/test/scala/keycloak/AdminConsoleSimulation.scala]: Moved.

**Review by mstruk — APPROVED** [review]

**Pull request merged**